pkg/config/settings/runtime_setting_enable_streamlogs.go

```diff
@@ -0,0 +1,80 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package settings
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/DataDog/datadog-agent/comp/core/config"
+	"github.com/DataDog/datadog-agent/pkg/config/model"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const (
+	// StreamLogsConfigKey is the key for the rc stream logs config.
+	StreamLogsConfigKey = "logs_config.streaming.enable_streamlogs"
+
+	// StreamLogRunTimeSettingName is the name of the runtime setting.
+	StreamLogRunTimeSettingName = "enable_streamlogs"
+
+	// StreamLogRuntimeDescription is the description of the runtime setting.
+	StreamLogRuntimeDescription = "Enable/disable remote config streamlogs at runtime. Possible values: true, false"
```

Comment:
Again, the description is misleading. The setting is controlled by remote config, but itself it enables inclusion of logs in the flare.

Not that there is anything wrong having a runtime setting for this, is it really necessary? RC can call `config.Set` directly, the only benefit this adds is ability to do `agent config set enable_streamlogs true`, which seems redundant given that the setting doesn't affect agent runtime unless flare is running, and an option to add logs to a flare already exists.